ch3-javascript_expressions&operators/script.js

```diff
@@ -43,3 +43,11 @@ console.log(`Increased Valued of x = ${xz}`);
 
 // postfix is not clear as of now so will  add later
 // prefix changes the value of the operand immediately when the increment or decreament operator is applied.
+
+// *****  3) Comparison Operator -  *****
+const ab = 12;
+const bc = "12";
+console.log(`12 is equals to 12? ${12 == 12}`); // here '==' is quality operator. which check weather 2 operand are qeual or not.
```

Comment:
Why the content inside "" does not prints in this statement - 

console.log("12 is not quals to 32" + 12 != 32);